bin/oneoff/data_fix/time_spent_milliseconds_to_seconds

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+# The time_spent field in the user_levels table has been
+# recording time in milliseconds. This will cause time_spent
+# to max out at ~25 days. Instead we want to record time in
+# seconds. This will cause time_spent to max out at ~68 years
+#
+# Relevant entries include:
+# SELECT *
+# FROM user_levels
+# WHERE time_spent IS NOT NULL
+#   AND time_spent > 0;
+#
+# time_spent should be transformed as such
+# (time_spent.to_f/1000).ceil.to_i
+
+# upper limit: 2811329000;
+# lower limit:   93221000;
+
+UserLevel.find_each(start: 93_221_000, finish: 2_811_329_000, batch_size: 5000).with_index do |user_level, index|
+  if index % 10000 == 0
+    puts "Starting transform of user_level with id #{user_level.id}"
+  end
+  if user_level.time_spent && user_level.time_spent > 0
+    user_level.time_spent = (user_level.time_spent.to_f / 1000).ceil.to_i
+    user_level.save(touch: false, validate: false)
```

Comment:
The ActiveRecord `update_column` method might be appropriate to use here.
https://apidock.com/rails/ActiveRecord/Persistence/update_columns